src/main/java/com/google/dicomwebfuse/entities/cache/Cache.java

```diff
@@ -205,30 +205,77 @@ public void setInstanceCommand(DicomPath dicomPath, Command command) throws Dico
         instanceContent.setCommand(command);
         break;
       default:
-        throw new DicomFuseException("Invalid dicom path level! " + dicomPath);
+        throw new DicomFuseException("Invalid dicom path level - " + dicomPath);
     }
   }
 
+  /**
+   * Checks that Dataset outdated or not.
+   *
+   * {@code Instant.now()} - Returns a value in microseconds in Java 9 and later, but returns
```

Comment:
This shouldn't be part of the Javadoc of this method... a user using this method doesn't need to know these details. Just put the comment inside the function definition.

Reply:
Done